addons/binding/org.openhab.binding.onebusaway/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.binding</groupId>
+    <artifactId>pom</artifactId>
+    <version>2.1.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.openhab.binding</groupId>
+  <artifactId>org.openhab.binding.onebusaway</artifactId>
+  <version>2.1.0-SNAPSHOT</version>
+
+  <name>OneBusAway Binding</name>
+  <packaging>eclipse-plugin</packaging>
+
+  <properties>
```

Comment:
I believe these are not required can you remove, I'll remove them from the others later on